### ISSUE TITLE
unobfuscated js

### ISSUE THREAD

**Issue by mte0** [thousandyearhog/WangDongStealer#1]
Hey man, cool project looks pretty sick, thanks for your work!

is there anyway you could upload/link an obfuscated version of the 2 JS files?

thanks in advance,
- mte

**Comment by annefrankfan** [comment]
I opened the solution file into visual studio code but it does nothing

**Comment by 9ggy** [comment]
Hello! I don't think that the owner will be posting unobfuscated versions of the JS files, sorry. Thankfully you can unobfuscate them yourself somewhere like https://deobfuscate.io/

The only issue here being you wont get intuitive/understandable variable or function names.

**Comment by thousandyearhog** [comment]
Why?

**Comment by annefrankfan** [comment]
I opened the sln file into VS and it doesn't do anything, am I doing something wrong, or is there another step.

**Comment by mte0** [comment]
@annefrankfan yeah the step you are missing is to stop skidding and learn how to code (or kys)

**Comment by mte0** [comment]
> Why?

idk, just not a fan of running obfuscated js 😭
@thousandyearhog

**Comment by annefrankfan** [comment]
Too much work, just tell me
